docs/source/querying/querybuilder/queryhelp.rst

```diff
@@ -317,73 +306,71 @@ Let's take an example that we had and add a few filters on the link::
         ],
         'filters':{
             'structure':{
-                'time':{'>': t},
                 'id':{'>': 50}
             },
             'relax--structure':{
-                'time':{'>': t},
                 'label':{'like':'output_%'},
             }
         },
         'project':{
             'relax--structure':['label'],
             'structure':['label'],
-            'relax':['label', 'state'],
+            'relax':['label', 'uuid'],
```

Comment:
I guess you could use 'process_state' instead? Also in an example above, where you changed 'state'.

Reply:
I just tried and this is what I got:
```
InputValidationError: process_state is not a column of aliased(DbNode)
Valid columns are:
id
uuid
node_type
process_type
label
description
ctime
mtime
attributes
extras
user_id
dbcomputer_id
```